### DISCUSSION TITLE
Is Support for hls video stream still been developed?

### DISCUSSION THREAD

**Discussion by ghost** [FDH2/UxPlay#360] [General]
The last commit for the branch video2 is 2 months ago

**Comment by fduncanh** [comment]
yes,   ~~in branch video3  (not yet functional).~~   work was postponed while h265 support was add to UxPlay-1.70.

Now in branch video4   (2024-11-11)

HLS  is next on this list, maybe in UxPay-1.71.

**Comment by fduncanh** [comment]
Now available  as UxPlay-1.71beta in  UxPlay  branch video4 https://github.com/FDH2/UxPlay/tree/video4

see #326    **PLEASE TEST**

You need the "-hls" option to activate HLS (HTTP Live Streaming) support.

* only YouTube HLS video is supported at present.

To clone using git (on Linux):

```
git clone http://github.com/FDH2/UxPlay
cd UxPlay
git checkout video4
mkdir build
cd build
cmake ..                  
make                        (on Windows with MSYS2, this becomes "ninja")
sudo make install           (on Windows "cmake --install . --prefix /mingw64")
```


```
